src/type/index.ts

```diff
@@ -0,0 +1,9 @@
+export type OperationKeyType = {
+  [key: string]: number;
+};
+
+export type CalculateParams = {
+  first: string;
+  second: string;
+  operator: string;
+};
```

Comment:
```suggestion
export type OperationKeyType = Record<string, number>;

export type CalculateParams = Record<'first' | 'second' | 'operator', string>;
```

이렇게 Record를 활용하는 방법도 있답니다!